tests/test_wps_multiple_outputs.py

```diff
@@ -10,7 +9,7 @@
 @pytest.fixture
 def resp():
     client = client_for(Service(processes=[MultipleOutputs()], cfgfiles=CFG_FILE))
-    datainputs = 'count={}'.format(5)
+    datainputs = f'count={5}'
```

Comment:
What's wrong with "count=5" ?

Reply:
This makes it pop. I think I just got carried away, will change.